roles/deploy_daskhub/files/config-gpu.yaml.template

```diff
@@ -0,0 +1,189 @@
+jupyterhub:
+  proxy:
+    https:
+      enabled: true
+      hosts:
+        - <Host Domain>
+      letsencrypt:
+        # Uncomment acmeserver for a test certificate
+        #acmeServer: "https://acme-staging-v02.api.letsencrypt.org/directory"
+        # This email gets warnings if auto-renewal fails
+        contactEmail: <Not Set>
+    # This can be generated with `openssl rand -hex 32`
+    secretToken: <Not Set>
+
+  singleuser:
+    defaultUrl: "/lab"
+    extraEnv:
+      DASK_GATEWAY__CLUSTER__OPTIONS__IMAGE: '{JUPYTER_IMAGE_SPEC}'
+    image:
+      name: <Image>  # Image to use for singleuser environment. Must include dask-gateway.
+      tag: <Tag>
+
+    storage:
+      # Required for PyTorch
+      extraVolumes:
+        - name: shm-volume
+          emptyDir:
+            medium: Memory
+        - name: training-materials
+          persistentVolumeClaim:
+            claimName: training-materials
+      extraVolumeMounts:
+        - name: shm-volume
+          mountPath: /dev/shm
+        - name: training-materials
+          mountPath: /mnt/materials
+
+      type: dynamic
+      capacity: 10Gi
+      dynamic:
+        storageClass: longhorn
+
+    # Default profile, this is what the placeholders will use
+    startTimeout: 1200 # seconds == 20 minutes
+    cpu:
+      limit: 12
+      guarantee: 1
+    memory:
+      limit: 1.8G
+      guarantee: 1.8G
+
+    profileList:
+      - display_name: "Dask GPU environment"
+        kubespawner_override:
+          tolerations:
+            - key: nvidia.com/gpu
+              operator: Equal
+              effect: NoSchedule
+          extra_resource_limits:
+            nvidia.com/gpu: "1"
```

Comment:
We don't want to give the user a GPU, only the workers should have these else it's the same as Jupyterhub

Reply:
Agreed that it's not ideal, but without this we cannot import libraries such as `cupy`, as it looks for an Nvidia GPU.